src/backend/tests/test_userpage.py

```diff
@@ -1,60 +1,137 @@
 """Test userpage.py"""
 import sqlite3
-from decorators import use_test
+from decorators import use_test, use_database
+from tests.mainpage_staging import MainPageStaging
 from pages.userpage import UserPage
-from pages.login import validate_phone
+from dataholders.apt import Apt
 
 
 class TestUserPage:
     """Test user page class"""
 
     username = "test_username"
-    alt_username = "alt_username"
-    userpage = UserPage(username)
     phone = "012-345-6789"
-    invalid_phone = "123-3421-322"
+    password = "newpassword1234"
+    email = "newemail@gmail.com"
+    userpage = None
+    main_page_staging = MainPageStaging()
 
-    @use_test
-    def test_valid_phone(self) -> bool:
-        """Test update_phone returns True and db entry is the same"""
-        assert validate_phone(self.phone) is True
-        assert self.userpage.update_phone(self.phone) is True
+    @use_database
```

Comment:
does this one have a `@use_test`? Otherwise it will write to db_prod

Reply:
ok will switch to use test

Reply:
Also `@use_test` should go before `@use_database` like 
`
@use_test
@use_database
do_stuff()
`

> ok will switch to use test

Reply:
I switch to use_test and everything fails now